src/sentry.js

```diff
@@ -3,7 +3,7 @@ import { Sentry } from 'react-native-sentry';
 
 import config from './config';
 
-if (config.enableSentry) {
+if (config.sentryKey != null) {
```

Comment:
This one is `!= null` and the others are `!== null`. Is there a reason to make them different?

Given that the type excludes `undefined` I'm inclined to make them all `!==`.

Reply:
No, this is just an alienism that happens not to have been caught by the linter. Fixed.